go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	knative.dev/eventing v0.39.3
 	knative.dev/pkg v0.0.0-20231023151236-29775d7c9e5c
 	knative.dev/serving v0.39.3
-	sigs.k8s.io/controller-runtime v0.15.2
+	sigs.k8s.io/controller-runtime v0.16.0
```

Comment:
I think we can bump to `0.16.5` in one shot. In theory, the biggest problem would be the incompatibility with the minor. The patch should introduce no breaking changes I guess.

Reply:
That's the plan, but for now I make the minimal change to see if the cache works.